helm-chart/ray-cluster/values.yaml

```diff
@@ -216,3 +216,4 @@ additionalWorkerGroups:
 service:
   # This is optional, and the default is ClusterIP.
   type: ClusterIP
+  headService: {}
```

Comment:
Add a comment and add this link: https://pkg.go.dev/k8s.io/Kubernetes/pkg/api/v1#Service

Reply:
Makes sense, thanks for the link. Done